src/mono/mono/mini/unwind.c

```diff
@@ -280,7 +280,7 @@ decode_sleb128 (guint8 *buf, guint8 **endbuf)
 	gint32 res = 0;
 	int shift = 0;
 
-	while (TRUE) {
+	for (;;) {
```

Comment:
same as the one above - `while (TRUE)` feels clearer to me.

Reply:
As mentioned above.